src/common/TextInput.tsx

```diff
@@ -68,6 +68,10 @@ const styles = StyleSheet.create({
     background: "transparent",
     borderRadius: "60px",
     alignSelf: "flex-start",
+    "::placeholder": {
+      color: "#FFFFFF",
+      textAlign: "center",
+    },

```

Comment:
This is the correct place to have the styling as opposed to the above comment. 

Reasoning: This is supposed to be a core component, if it should be a certain way, then it should be on the core component. Reduce random extra code